.github/workflows/cypress.yml

```diff
@@ -7,14 +7,14 @@ on:
     paths-ignore:
       - 'docs/**'
       - '*.md'
```

Comment:
this should be `**.md` right?

Reply:
Yes, `**.md` would match any Markdown file in the project